config/forms/2021_2022.json

```diff
@@ -1796,18 +1796,21 @@
           "depends_on": { "about_this_log": "completed" },
           "pages": {
             "net_income": {
-              "header": "",
+              "header": "Household’s combined income",
               "description": "",
               "questions": {
                 "net_income_known": {
                   "check_answer_label": "Income known",
-                  "header": "Do you know the tenant and their partner’s net income?",
+                  "header": "Do you know the household’s combined income after tax?",
+                  "guidance_partial": "what_counts_as_income",
                   "hint_text": "",
                   "type": "radio",
                   "answer_options": {
-                    "0": "Yes",
-                    "1": "No",
-                    "2": "Tenant prefers not to say"
+                    "0": "Yes – the household has a weekly income",
+                    "1": "Yes – the household has a monthly income",
+                    "2": "Yes – the household has a yearly income",
+                    "divider_a": true,
+                    "3": "Tenant prefers not to say"
```

Comment:
We got some inferred variables that rely on existing values like 
`self.incref = 1 if net_income_known == "Prefer not to say"` in `models/case_log.rb:217` (of course this would've failed before too, but might be a good place to fix)
Also same file lines `319` `332`

Reply:
Good catch. Have updated but this seems like a good argument for switching to using the ids rather than values for comparisons everywhere.